packages/fx-core/src/plugins/solution/fx-solution/index.ts

```diff
@@ -2,6 +2,7 @@ export * from "./solution";
 export * from "./constants";
 import "./v2/solution";
 import "./v3/solution";
+export * from "./question";
```

Comment:
is this used?

Reply:
Yes, use the `AzureSolutionQuestionNames` in `fx-solution`.